lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -144,51 +152,84 @@ def _get_config(self, config_dict):
             he_proxy_eth_mac=he_proxy_eth_mac)
 
     def initialize_on_connect(self, datapath):
-        self.delete_all_flows(datapath)
-        self._install_default_ingress_flows(datapath)
-        self._install_default_egress_flows(datapath)
-        self._install_default_middle_flows(datapath)
+        self._datapath = datapath
         self._setup_non_nat_monitoring(datapath)
 
+    def _install_default_flows_if_not_installed(self, datapath,
```

Comment:
functionality to generate flaw table diff can be moved to common lib in RestartMixin.

Reply:
Yeah good points I'll just keep the msg creation controller side and pull out all these into the restart mixin next diff